src/component/audit/ErrorLogViewer.tsx

```diff
@@ -43,9 +43,15 @@ const ErrorLogViewer: React.FC = () => {
           {errors.map((item) => {
             let error = item.error;
             if (error.messageId) {
-              error = Object.assign({}, error, {
-                message: i18n(error.messageId),
-              });
+              if (error.values && Object.keys(error.values).length > 0) {
+                error = Object.assign({}, error, {
+                  message: formatMessage(error.messageId, error.values),
```

Comment:
What if you used `formatMessage` even when error.values is empty? Wouldn't it behave the same way as `i18n`?

Reply:
Yes, it behaves the same way.